### PR TITLE
feat: Add a form to fr_FR i18n

### PR DESCRIPTION
**Pull request by Mouerr**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Perfermance optimization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [x] Other (about what?)

### 🔗 Related issue link

https://github.com/ant-design/ant-design/issues/23369

### 💡 Background and solution

I think the validation messages for form in the fr_FR could be the same as the fr_BE

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |add fr_FR localisations for form|
| 🇨🇳 Chinese |           |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: components/locale/fr_FR.tsx [hunk 1]
```diff
@@ -57,6 +60,55 @@ const localeValues: Locale = {
...
+      array: {
+        len: 'Le nombre d\'éléments ${label} doit être égal à ${len}',
+        min: 'Le nombre d\'éléments ${label} doit être supérieur ou égal à ${min}',
+        max: 'Le nombre d\'éléments ${label} doit être inférieur ou égal à ${max}',
+        range: 'Le nombre de ${label} doit être compris entre ${min} - ${max}',
+      },
+      pattern: {
+        mismatch: '${label} ne correspond pas au modèle ${pattern}',
+      },
+    }
```
Comment (yoyo837) [inline components/locale/fr_FR.tsx:110]:
```suggestion
    },
```

File: components/locale/fr_FR.tsx [hunk 2]
```diff
@@ -57,6 +60,55 @@ const localeValues: Locale = {
...
+  }
```
Comment (yoyo837) [inline components/locale/fr_FR.tsx:111]:
```suggestion
  },
```

**Comment by yoyo837** [comment]
Pls update the snapshot.

**Comment by afc163** [comment]
Please rebase master.

**Comment by afc163** [comment]
Please rebase master.

**Comment by Mouerr** [comment]
i did it but should i run npm test -- -u again to update snapshots?

**Comment by yoyo837** [comment]
Please rebase master again.

**Comment by afc163** [comment]
closing for no response

**Pull request closed without merging**